### ISSUE TITLE
DXVK launch option

### ISSUE THREAD

**Issue by DealsBeam** [JuneStepp/OneLauncher#66]
Would be great if you could add a feature to launch the game with DXVK on Windows https://github.com/doitsujin/dxvk

**Comment by JuneStepp** [comment]
Interesting idea. Have you tried it before? If so, how helpful was it?

**Comment by DealsBeam** [comment]
Yes, I've tried it and I'm getting increased performance, feel free to check it out yourself.

I'm surprised Standing Stone Games have not officially supported it.

**Comment by JuneStepp** [comment]
I don't use Windows, but OneLauncher has used DXVK on Linux/Mac for years. I've added looking into it on Windows to the roadmap.

**Comment by FibroJedi** [comment]
Sorry to resurrect this - but if the Linux version _is_ using DXVK, is it possible to add options to parse the DXVK_HUD environment variable please? I am trying to see how to improve LOTRO's performance in certain situations, so wanted to have DXVK_HUD="gpuload,fps,compiler" for example.

In other apps (e.g. through Lutris) you can set that, but I don't think I can do it through OneLauncher, even with advanced settings?

Thanks!
